File: Lista3/zad1/Kolo.java
public class Kolo extends Figura
{
    double promien;

    public Kolo(double promien)
    {
        this.promien = promien;
    }

    public double obliczObwod()
    {
        return 2 * Math.PI * promien;
    }

    public double obliczPole()
    {
        return Math.PI * promien * promien;
    }

    public String podajNazwe()
    {
        return "Koło";
    }
}

File: Lista3/zad2/Kolo.cpp
#include "Kolo.hpp"
#include <math.h>
#include <string>
// I didnt find PI :(

Kolo::Kolo(double promien)
{
    r = promien;
}

double Kolo::obliczObwod()
{
    return r*3.14*2;
}

double Kolo::obliczPole()
{
    return r*r*3.14;
}

std::string Kolo::podajNazwe()
{
    return "Kolo";
}